app/podcast/page.tsx

```
@@ -7,6 +7,8 @@ import PodcastPlayer from '@/components/podcast/PodcastPlayer';
 import PlatformBadges from '@/components/podcast/PlatformBadges';
 
 export const metadata = { title: 'Podcast | CloudFix', description: 'Tune into AWS cost optimization conversations with CloudFix and guests.' };
+export const revalidate = 21600; // 6 hours ISR
+export const revalidate = 21600; // 6 hours ISR for podcast listing
```

Biome (2.1.2)

[error] Shouldn't redeclare 'revalidate'. Consider to delete it or rename it.

'revalidate' is defined here:


(lint/suspicious/noRedeclare)